.azp/test.yml

```diff
@@ -0,0 +1,18 @@
+trigger: none
+
+pr:
+  - v3-api
+
+schedules:
+- cron: "0 2 * * *"
+  displayName: Daily 10PM build
+  # 2am UTC is 10PM eastern time
+  branches:
+    include:
+    - "*"
```

Comment:
I am assuming the `pr` above means that this will still only run on that PR?

Reply:
`test.yml` runs on PRs into the `v3-api` branch, and nightly on any branches (whether or not they are part of PRs) that have changes from the night before (`always: false`). This is the same setup I use for gsd, fresnel, and hoomd and is the result of a limitation of the AZP scheduling system. AZP schedules branches and PRs separately and you can't exclude branches that are also part of PRs. I use the nightly builds for branches so that the tests don't run twice for every commit (once for the PR and once for the branch) and so that developers can get some feedback on a branch if they wish before submitting a PR.

`compatibility.yml` exists to implement the daily HOOMD master branch compatibility check we discussed. The finest granularity for notifications that the slack notification allows is by pipeline (top level .yml) file. If the compatibility checks were scheduled in `test.yml` we would also get notifications for every PR and branch. The schedule here only runs on the `v3-api` branch and runs every night even when there were no changes to the branch (`always: true`). The timing is set earlier by an hour so that these tests have a chance to run before all the other HOOMD validation tests run. `pr` and `trigger` are both `none` here so no builds will be triggered by commits.